index.ts

```diff
@@ -94,8 +94,39 @@ const colorSet: IColorSet = {
 
 generateTheme('Sapphire', colorSet, path.join(__dirname, 'theme.json'));
 
-colorSet.base.background = '#242d3d'
-generateTheme('Sapphire (Bright)', colorSet, path.join(__dirname, 'theme-bright.json'));
+const brightColorSet: IColorSet = {
+  base: {
+    ...colorSet.base,
+    background: '#242d3d',
+  },
+  syntax: {
+    ...colorSet.syntax,
+    comment: '#fff6'
+  },
+  ui: {
+    ...colorSet.ui
+  },
+  terminal: {
+    ...colorSet.terminal
+  },
+  overrides: {
+    ...colorSet.overrides,
+
+    'diffEditor.insertedTextBackground': '#0f02',
+    'diffEditor.removedTextBackground': '#f003',
+    'diffEditor.insertedTextBorder': '#0004',
+    'diffEditor.removedTextBorder': '#0004',
+
+    'editor.background': '#2a3447',
+    'editor.selectionBackground': '#fff3',
+
+    'editorWhitespace.foreground': '#fff2',
```

Comment:
Ignore this change. I need to remove it and rebuild the theme. This should be a local-to-me change only.